### PR TITLE
Faster printing.

### PR DESCRIPTION
**Pull request by JackDainzh**
This is the initial pull-request for the faster version. Tested it only on 1.19 and haven't tested it out on 1.19.2 as I didn't have a need for that. Feel free to modify it and test it, I think it should work the same.

### CODE REVIEW

**Comment by momokinou** [comment]
is it possible to merge please? @aleksilassila

**Comment by aleksilassila** [comment]
> is it possible to merge please? @aleksilassila

Right sorry, I'll make some reviews, if you could solve first

File: README.md [hunk 1]
```diff
@@ -1,37 +1,2 @@
-Litematica Printer
```
Comment (aleksilassila) [inline README.md:1]: Please don't include this in the merge

**Review by aleksilassila — CHANGES_REQUESTED** [review]
Could you use the same line endings that the rest of the project uses (LF, hopefully, if I have set things up correctly) and also cherry pick your commits to printing_1.18 and printing_1.17 (or that's what I do, lmk if there's a better way lol) to keep the versions consistent code-wise, and then pull requests for those branches also. Hopefully you don't actually need to change your code between the versions, let me know if you do. For that I have the [Implementation.java](https://github.com/aleksilassila/litematica-printer/blob/printing/src/main/java/me/aleksilassila/litematica/printer/interfaces/Implementation.java) file, that should contain all the version specific code.

Hit me up if you need help with any of this :)

**Comment by momokinou** [comment]
@JackDainzh :D

**Comment by JackDainzh** [comment]
ah, sorry, for now I am not so keen on resolving stuff, I just did the base, not to familiar with github and versions codes in general. If I have free time someday, I might pull this off tho.

**Comment by aleksilassila** [comment]
I included your solution in the printer rewrite I'm currently doing, thank you for your solution. Gonna merge by hand since the files changed so drastically.

**Pull request closed without merging**